addons/binding/org.openhab.binding.mihome/ESH-INF/thing/sensor_smoke.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="mihome" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="smoke">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="bridge" />
+		</supported-bridge-type-refs>
+
+		<label>Xiaomi Smoke Sensor</label>
+		<description>New, yet unconfirmed device</description>
+
+		<channels>
+			<channel id="alarm" typeId="smoke_alarm"></channel>
+			<channel id="status" typeId="status-string"></channel>
```

Comment:
Does it share exactly the same statuses with the gas sensor? If not, better define a dedicated type for it.